sql/mob_spawn_points.sql

```diff
@@ -56543,10 +56543,10 @@ INSERT INTO `mob_spawn_points` VALUES (17428485,'Bouncing_Ball','Bouncing Ball',
 
 INSERT INTO `mob_spawn_points` VALUES (17428486,'Bonze_Marberry','Bonze Marberry',4,340.103,-0.001,222.666,127);
 INSERT INTO `mob_spawn_points` VALUES (17428487,'Trompe_LOeil','Trompe L\'Oeil',5,80.031,-0.500,-38.824,197);
-INSERT INTO `mob_spawn_points` VALUES (17428488,'Cook_Solberry','Cook Solberry',6,0.000,0.000,0.000,0);
-INSERT INTO `mob_spawn_points` VALUES (17428489,'Cook_Nalberry','Cook Nalberry',7,0.000,0.000,0.000,0);
-INSERT INTO `mob_spawn_points` VALUES (17428490,'Cook_Minberry','Cook Minberry',8,0.000,0.000,0.000,64);
-INSERT INTO `mob_spawn_points` VALUES (17428491,'Cook_Fulberry','Cook Fulberry',9,0.000,0.000,0.000,0);
+INSERT INTO `mob_spawn_points` VALUES (17428488,'Cook_Solberry','Cook Solberry',6,-144.0,0.000,-97.0,231);
+INSERT INTO `mob_spawn_points` VALUES (17428489,'Cook_Nalberry','Cook Nalberry',7,-130.0,0.000,-101.0,186);
+INSERT INTO `mob_spawn_points` VALUES (17428490,'Cook_Minberry','Cook Minberry',8,-142.0,0.000,-101.0,234);
+INSERT INTO `mob_spawn_points` VALUES (17428491,'Cook_Fulberry','Cook Fulberry',9,-132.0,0.000,-97.0,123);
```

Comment:
These are probably fine but you can get the positions by looking at the spawn packets with VieweD

Reply:
I noted in the comments above but it looks like when this specific capture was taken signing into another account lost the packet captures, I might be missing something but I don't believe there's packet data captured for this section - the packet viewer/full.log file ends at 16:21:58 for me, these mobs were first spawned at 16:55:04 🤷‍♂️